core/tx_pool.go

```diff
@@ -238,6 +239,7 @@ func (pool *TxPool) SetGasPrice(price *big.Int) {
 	log.Info("Transaction pool price threshold updated", "price", price)
 }
 
+// State returns the state of TxPool
```

Comment:
Please add a period to the end of the comment, and write out `transaction pool`, not `TxPool`.